src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -654,4 +653,25 @@ private static File getCopyNumberSegmentsFile(final File pythonSegmenterOutputPa
         }
         return unsortedIntervalCollectionsFromModels;
     }
+
+    /**
+     * Validate that the union of shard's interval lists does not have singleton intervals, i.e. intervals that
```

Comment:
`Validate that the concatenation of the sharded interval lists does not have singleton intervals...`, perhaps?

Reply:
Changed